test/models/fake_auth_test.rb

```diff
@@ -6,7 +6,7 @@ class FakeAuthTest < ActiveSupport::TestCase
   test 'fakeauth disabled' do
     ClimateControl.modify(
       FAKE_AUTH_ENABLED: 'false',
-      HEROKU_APP_NAME: 'thesis-dropbox-pr-123'
+      HEROKU_APP_NAME: 'thesis-submit-pr-123'
```

Comment:
Would you mind adding the old tests back as well as these updated ones? I know we'll rip some of this out when we retire the old pipeline, for now showing that both the old and new patters are working explicitly in tests with that regex would be great.

Also, I now wish we controlled fake auth patterns and pr build patterns in ENV but let us not go down that path now even if it would feel good...

Reply:
I think I've now done this - although for some reason the diff on this screen isn't looking right to me. The source code itself at https://github.com/MITLibraries/thing/blob/etd-148-fake-auth/test/models/fake_auth_test.rb#L6-L34 looks correct, with tests for both old and new review app names, but the combined diff looks like it isn't displaying all the relevant lines.